x-pack/plugins/maps/public/classes/styles/vector/vector_style.tsx

```diff
@@ -703,6 +717,15 @@ export class VectorStyle implements IVectorStyle {
       : (this._iconStyleProperty as StaticIconProperty).getOptions().value;
   }
 
+  getIconMeta(symbolId: string | undefined) {
```

Comment:
This is typed differently then getIconMeta in interface. Type as `getIconMeta(symbolId: string)` and add logic to callers to not call getIconMeta if symbolId is undefined